specification/translation/Azure.AI.DocumentTranslation/client.tsp

```diff
@@ -5,6 +5,10 @@
 import "./main.tsp";
 import "@azure-tools/typespec-client-generator-core";
 
+// model overrides
+@@access(DocumentTranslation.TranslationStatus, Access.internal, "python");
+@@access(DocumentTranslation.DocumentStatus, Access.internal, "python");
+
```

Comment:
Can you move these to the bottom of the file?

Reply:
Moved